microsoft-edge/webview2/concepts/data_privacy.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Data & Privacy in WebView2
+description: Overview of WebView2 Data Collection and Practices. 
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/03/2023
+---
+# Data & Privacy in WebView2 
+
+## Background
+The WebView2 team tracks data to help understand and prioritize our customers' requirements, diagnose and debug issues, and overall improve WebView2 as a product.
+Developers consent to this data collection in the WebView2 Runtime Terms and Conditions License, or in the Windows license. Additionally, we follow the standards outlined in the [Microsoft Edge Privacy Whitepaper](https://learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/).
+
+In general, all of our data collection goes through the same strict standards as Microsoft Edge. The main telemetry sources are Chromium and Edge telemetry infrastructure, Windows data reporting, and Watson (Microsoft Internal) infrastructure to collect crash dumps.
+With any data the WebView2 team collects, we comply/conform with the laws and regulations about best privacy practices, and have guardrails to protect against privacy issues.
+
+## Optional Diagnostic Data
+
+WebView2 and Edge respect the Windows Optional Diagnostic Data toggle, so end users have control over any optional data WebView2 collects. This toggle can only be modified by the end user. The only mandatory data that is collected is the [required events](https://learn.microsoft.com/en-us/windows/privacy/required-windows-11-diagnostic-events-and-fields) which are required from any app running on Windows.
+
+WebView2 is considered a Windows component, and the data collection consent is governed by Windows Diagnostic data setting on Windows 10 & 11 as a centralized switch.

```

Comment:
```suggestion
WebView2 is considered a Windows component, and the data collection consent is governed by Windows Diagnostic data setting on Windows 10 & 11 as a centralized switch.
```
Global: change & to and unless there's a special reason eg UI or name officially includes &

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/ampersand -- "Don't use & in place of and in text or headings unless you're referring to the use of the symbol in UI."